src/components/plots/barPlot.ts

```diff
@@ -319,9 +319,8 @@ export module Plot {
         attrToProjector[secondaryAttr] = (d: any, i: number, u: any, m: PlotMetadata) =>
           positionF(d, i, u, m) - widthF(d, i, u, m) * this._barAlignmentFactor;
       } else {
```

Comment:
We should be able to merge the two halves of this if we only have one mode right?